<launch>
    <arg name="joint_state_topic_name" default="/joint_states_host" />
    <arg name="curobo_config_file_path" default="/home/yilin/dataset/robot_description/a1arm_description/a1arm.yml" />
    <arg name="arm_joint_position_control_topic_name" default="/arm_joint_target_position"/>
    <arg name="gripper_joint_position_control_topic_name" default="/gripper_position_control_host"/>

    <arg name="episode_raw_dir" default="/home/yilin/simple_robot_ws/outputs/train/2024-11-27/23-32-32_real_world_act_default/checkpoints/last/pretrained_model" />

    <node name="replay_episode" pkg="simple_hand_teleop" type="replay_episode.py" output="screen">
        <param name="joint_state_topic_name" value="$(arg joint_state_topic_name)" />
        <param name="curobo_config_file_path" value="$(arg curobo_config_file_path)" />
        <param name="arm_joint_position_control_topic_name" value="$(arg arm_joint_position_control_topic_name)" />
        <param name="gripper_joint_position_control_topic_name" value="$(arg gripper_joint_position_control_topic_name)" />

        <param name="episode_raw_dir" value="$(arg episode_raw_dir)" />
    </node>
</launch>